variables.md

```diff
@@ -174,6 +174,7 @@ SES5_CEPH_QA_HEALTH_OK | string | | URL for repo containing ceph-qa-health-ok pa
 SKIP_CERT_VALIDATION | boolean | false | Enables linuxrc parameter to skip certificate validation of the remote source, e.g. when using self-signed https url.
 SET_CUSTOM_PROMPT | boolean | false | Set a custom, shorter prompt in shells. Saves screen space but can take time to set repeatedly in all shell sessions.
 SLE_PRODUCT | string | | Defines SLE product. Possible values: `sles`, `sled`, `sles4sap`. Is mainly used for SLE 15 installation flow.
+SLURM_VERSION | string | | if given it installed versioned Slurm, otherwise it uses the base version. ex 23_02.
```

Comment:
Current sentence doesn't make much sense. I would say `Defines slurm version (23_02) for installation. `

Reply:
I add this and modified the rest. i hope it makes more sense